MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,572 @@
+imageEditor.Source = ImageSource.FromFile("image.png");
+imageEditor.AutoGenerateToolbarItems = false;
+ImageEditorToolbar editorToolbar = new ();
+editorToolbar.Orientaion = ToolbarOrientation.Vertical;
+editorToolbar.Position = ToolbarPosition.End;
+editorToolbar.ToolbarItems = new List<IImageEditorToolbarItem>()
+{
+    new ImageEditorToolbarItem(){ Name = "Text",
+        SubToolbars = new List<ImageEditorToolbar>()
+        {
+            new ImageEditorToolbar()
+            {
+                ToolbarItems = new List<IImageEditorToolbarItem>()
+                {
+                   new ImageEditorToolbarItem(){ Name = "AddText"},
+                   new ImageEditorToolbarItem(){ Name = "TextColor"},
+                   new ImageEditorToolbarItem(){ Name = "Delete"}
+                },
+            }
+        }
+    },
+    new ImageEditorToolbarItem(){ Name = "Shape"},
+    new ImageEditorToolbarItem(){ Name = "Pen"}
+};
+
+imageEditor.Toolbars.Add(editorToolbar);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add an item with the existing toolbar.
+
+Creating custom toolbars in SfImageEditor provides you with complete control over the layout and content of your app's toolbar. 
+
+{% tabs %}
+
+{% highlight XAML %}
+
+<imageEditor:SfImageEditor Source="image.png">
+    <imageEditor:SfImageEditor.Toolbars>
+        <imageEditor:ImageEditorToolbar>
+            <imageEditor:ImageEditorToolbar.ToolbarItems>
+                <imageEditor:ImageEditorToolbarItem Name="Browse" />
+                <imageEditor:ImageEditorToolbarItem Name="Save" />
+            </imageEditor:ImageEditorToolbar.ToolbarItems>
+        </imageEditor:ImageEditorToolbar>
+        <imageEditor:ImageEditorToolbar>
+            <imageEditor:ImageEditorToolbar.ToolbarItems>
+                <imageEditor:ImageEditorToolbarItem Name="Text" />
+                <imageEditor:ImageEditorToolbarItem Name="Shape" />
+                <imageEditor:ImageEditorToolbarItem Name="Pen" />
+            </imageEditor:ImageEditorToolbar.ToolbarItems>
+        </imageEditor:ImageEditorToolbar>
+    </imageEditor:SfImageEditor.Toolbars>
+</imageEditor:SfImageEditor>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+ImageEditorToolbar editorToolbar = new ();
+editorToolbar.ToolbarItems = new List<IImageEditorToolbarItem>()
+{
+    new ImageEditorToolbarItem(){ Name = "Browse"},
+    new ImageEditorToolbarItem(){ Name = "Save"}
+};
+
+ImageEditorToolbar annotationToolbar = new ();
+editorToolbar1.ToolbarItems = new List<IImageEditorToolbarItem>()
+{
+    new ImageEditorToolbarItem(){ Name = "Text"},
+    new ImageEditorToolbarItem(){ Name = "Shape"},
+    new ImageEditorToolbarItem(){ Name = "Pen"}
+};
+
+imageEditor.Toolbars.Add(editorToolbar);
+imageEditor.Toolbars.Add(annotationToolbar);
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add new custom toolbar 
+
```

Comment:
## Explicitly create Toolbars
The ImageEditor auto-generates the toolbars, You can explicitly add the Toolbars by adding the Toolbars manually and you need to change the AutoGenerateToolbarItems property to false.

Reply:
Updated